packages/storybook/cxl-lumo-styles/vaadin-tabs/vaadin-tabs[theme=cxl-tabs-slider].stories.js

```diff
@@ -0,0 +1,87 @@
+import { html } from 'lit-html';
+import '../../../cxl-ui/src/components/cxl-card';
+import archiveData from '../../cxl-ui/cxl-vaadin-accordion/theme=cxl-archive.data.json';
+
+export default {
+  title: 'CXL Lumo Styles/Elements/vaadin-tabs',
+};
+
+export const Default = () => html`
+  <style>
+    html {
+      font-family: var(--lumo-font-family);
+    }
+
+    .entry {
+      transition: all 0.2s ease-in;
+    }
+
+    img {
+      display: block;
+    }
+
+    .testimonials-container {
```

Comment:
Where is this class used?

Name implies it's invented, so likely drop candidate?

Reply:
Removed